addons/sap-success-factors/install.js

```diff
@@ -0,0 +1,142 @@
+/*
+This is the installation logic of the Lifecycle Manager. 
+*/
+
+const Fs = require('fs');
+const Sdk = require('@fusebit/add-on-sdk');
+const Superagent = require('superagent');
+
+const getTemplateFiles = (fileNames) =>
+  fileNames.reduce((a, c) => {
+    a[c] = Fs.readFileSync(__dirname + `/template/${c}`, { encoding: 'utf8' });
+    return a;
+  }, {});
+
+module.exports = async (ctx) => {
+  let storageSpecification = await Sdk.createStorage(ctx);
+
+  // Create the Add-On Handler
+  try {
+    await Sdk.createFunction(ctx, {
+      configurationSerialized: `# Add-on configuration settings
+${Object.keys(ctx.body.configuration)
+  .sort()
+  .map((k) => `${k}=${ctx.body.configuration[k]}`)
+  .join('\n')}
+
+  # Storage configuration settings
+${Object.keys(storageSpecification)
+  .sort()
+  .map((k) => `${k}=${storageSpecification[k]}`)
+  .join('\n')}
+`,
+      nodejs: {
+        files: getTemplateFiles([
+          'index.js',
+          'package.json',
+          'success-factors.js',
+          'client/candidate/BatchRequest.js',
```

Comment:
Were these not already published in a package on npm?

Reply:
Nope, see my comment in the description

Reply:
Well I guess it'll be easier to embed the auth and url aspects into it this way.

Though - is there a reason you're not publishing an NPM package for it?  Speed-to-delivery is a fine reason, just making sure there's not a technical one.

Can you add the comments at the top of this PR into a README or something somewhere in the repo, for the next person to come along and wonder what those files are for?  Or even better, the incantation to generate them, perhaps even as a script in package.json?